Content.Shared/Roles/JobPrototype.cs

```diff
@@ -41,18 +44,36 @@ public sealed partial class JobPrototype : IPrototype
         [ViewVariables(VVAccess.ReadOnly)]
         public string? LocalizedDescription => Description is null ? null : Loc.GetString(Description);
 
+        /// <summary>
+        ///     Requirements for the job.
+        /// </summary>
         [DataField, Access(typeof(SharedRoleSystem), Other = AccessPermissions.None)]
         public HashSet<JobRequirement>? Requirements;
 
+        /// <summary>
+        ///     When true - the station will have anouncement about arrival of this player.
+        /// </summary>
         [DataField("joinNotifyCrew")]
         public bool JoinNotifyCrew { get; private set; } = false;
 
+        /// <summary>
+        ///     When true - the player will recieve a message about importancy of their job.
+        /// </summary>
         [DataField("requireAdminNotify")]
         public bool RequireAdminNotify { get; private set; } = false;
 
+        /// <summary>
+        ///     Should this job appear in preferences menu?
+        /// </summary>
         [DataField("setPreference")]
         public bool SetPreference { get; private set; } = true;
 
+        /// <summary>
+        ///     Should the selected traits be applied for this job?
+        /// </summary>
+        [DataField("applyTraits")]
```

Comment:
Not forcing a change of the other fields, but the new ones shouldn't have it.
```suggestion
        [DataField]
```
this makes no sense, since now if no DataField name is specified, it is automatically taken from the variable name, with a small letter